rustls/examples/internal/bench_impl.rs

```diff
@@ -363,24 +502,18 @@ fn make_client_config(
     cfg
 }
 
-fn apply_work_multiplier(work: u64) -> u64 {
```

Comment:
I don't see a strong justification for this change in this commit... Maybe move it to a separate commit if the `Options` structure will be helpful later on?

Reply:
This was necessary because `apply_work_multiplier` before used the "ambient" environment, but now the value comes from clap and needs to be passed down from there. But there's a degree of looking ahead in putting it in the `Options` struct.